src/Misc/InstallAgentPackage.template.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It feels like these probably belong under the `.azure-pipelines` directory, right? They're not really src code

Reply:
I'm not sure anything under Misc is really "source code". I don't have a strong preference where they live.